lib/screens/play/play_page.dart

```diff
@@ -23,6 +24,14 @@ class PlayPage extends StatelessWidget {
             child: Consumer<GameState>(builder: (context, state, _) {
               if (state.scrying.isNotEmpty) {
                 return ScryingPage(state);
+
+                /* TODO - game end stuff - new screen required?
+              } else if (state.result == GameResult.WIN ) {
+                return GameEndPage(state);
+              } else if (state.result == GameResult.LOSE ) {
+               return GameEndPage(state);
+                */
```

Comment:
Yep, something like this sounds reasonable to me. I guess we could start off with:

```
else if (state.result != null) {
    return GameEndPage(state.result);
}
```

...and then we can tease apart a separate `GameWinPage` / `GameLosePage` if that page grows too much (but I doubt it would - it's just going to be a message and a couple of buttons either way I would've thought :shrug: )